seed/challenges/intermediate-bonfires.json

```diff
@@ -606,10 +606,10 @@
         "smallestCommons([1,5]);"
       ],
       "tests": [
-        "expect(smallestCommons([1,5])).to.be.a('number');",
-        "expect(smallestCommons([1,5])).to.equal(60);",
-        "expect(smallestCommons([5,1])).to.equal(60);",
-        "expect(smallestCommons([1,13])).to.equal(360360);"
+        "assert.deepEqual(typeof(smallestCommons([1, 5])), \"number\", \"The result should be a number\");",
```

Comment:
This could use [`assert.isNumber`](http://chaijs.com/api/assert/#isNumber) instead.